docker/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to the docker containers will be documented in this file.
 ### 2022-11-04
   - Fixed `zap-x.sh` to return the exit code from `zap.sh` instead of `rm -f`
 
+### 2022-11-01
```

Comment:
This needs to be updated.

Reply:
> This needs to be updated and moved.

Just updated the changelog and rebased to get rid of the extra commits. thanks!